diff --git a/node_modules/react-native-progress/Bar.js b/node_modules/react-native-progress/Bar.js
index dd8e877..f5e33b0 100644
--- a/node_modules/react-native-progress/Bar.js
+++ b/node_modules/react-native-progress/Bar.js
@@ -136,7 +136,7 @@ export default class ProgressBar extends Component {
       borderWidth,
       borderColor: borderColor || color,
       borderRadius,
-      overflow: 'hidden',
+      overflow: 'visible',
       backgroundColor: unfilledColor,
     };
     const progressStyle = {
@@ -171,8 +171,18 @@ export default class ProgressBar extends Component {
         onLayout={this.handleLayout}
         {...restProps}
       >
-        <Animated.View style={progressStyle} />
-        {children}
+        <Animated.View style={progressStyle}>
+        <View
+                style={{
+                  position: 'absolute',
+                  backgroundColor: color,
+                  height: 12,
+                  width: 12,
+                  borderRadius: 10,
+                  top: -5,
+                  alignSelf: 'flex-end',
+                }}></View>
+          </Animated.View> 
       </View>
     );
   }
